private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -248,6 +250,12 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		"",
 		`The file or data to use for configuration.`,
 	)
+	flagSet.StringSliceVar(
```

Comment:
Is this what we want to do? We have plans to move more into the `buf.gen.yaml` into the future, this is going in the opposite direction. This feels like a major UI change